src/features/import-maps.js

```diff
@@ -33,7 +33,22 @@ function processScripts () {
       script.sp = true;
       if (!script.src)
         return;
-      System.import(script.src.slice(0, 7) === 'import:' ? script.src.slice(7) : resolveUrl(script.src, baseUrl));
+      System.import(script.src.slice(0, 7) === 'import:' ? script.src.slice(7) : resolveUrl(script.src, baseUrl)).catch(function (e) {
+        // if there is a script load error, dispatch an "error" event
+        // on the script tag.
+        if (e.message.indexOf('https://git.io/JvFET#3') > -1) {
+          var event
+          // IE 11 throw when callin new Event()
+          try {
+            event = new Event('error')
```

Comment:
I understand `createEvent` is deprecated, but if it doesn't warn, and it works in all browsers, then lets just stick with it for the whole API to save bytes unless there is a good reason to use both approaches.

Reply:
Good idea, done in d4c0fec, works on Chrome and Firefox at least. Not tested on IE11.